### PR TITLE
handle missing package config files

### PR DESCRIPTION
**Pull request by jakemac53**
Should fix an internal breakage with the latest release

### CODE REVIEW

**Review by natebosch — APPROVED** [review]

**Review by kevmoo — APPROVED** [review]

File: pkgs/test_core/lib/src/util/package_config.dart
Comment (kevmoo) [inline pkgs/test_core/lib/src/util/package_config.dart:35]:
```suggestion
  } on StateError {
```

Doesn't this work? 🤷 

Not a big deal AT ALL

Comment (jakemac53) [reply pkgs/test_core/lib/src/util/package_config.dart:35]: I think that is fine, but I don't want to wait for CI again as I want to get this merged internally to unblock some folks. I will follow up with a fix :)

Comment (jakemac53) [reply pkgs/test_core/lib/src/util/package_config.dart:35]: https://github.com/dart-lang/test/pull/2159

**Pull request merged**
